tests/test_crossover_duplication.py

```
@@ -0,0 +1,190 @@
+"""
+Tests for semantic crossover duplication bug.
+"""
+
+import pytest
+from unittest.mock import Mock, AsyncMock, patch
```

Ruff (0.12.2)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)